src/environment/set-env.ts

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+import { environment } from "./env";
+
+export const setDefaultEnvironment = () => {
+  Object.entries(environment).forEach(([key, value]) => {
+    // load the defaults first
+    process.env[key] = value;
+  });
+
+  // override from local .env file if anything is present
+  require("dotenv").config({ override: true });
```

Comment:
This part might be doing nothing, because env vars are loaded at run time, when a script that uses this function is called, all the `process.env` vars are mapped, then we override those with the defaults above in the `Object.entries` loop